resources/views/page.blade.php

```diff
@@ -1,5 +1,11 @@
 @extends('adminlte::master')
 
+@if(config('adminlte.layout_topnav') || View::getSection('layout_topnav'))
```

Comment:
This should not be directed by layout-topnav, you can use one of the two classes with any layout option enabled.

Reply:
Ok, I'm going to give a review. That change was based on the examples shown by the AdminLTE site. So, you say any of the classes "container" or "container-fluid" can be used independent of the layout, but is one of they always required?

Reply:
I have made a new commit that take this into considerations, now `$def_container_class` will be used only as a default value, when the configuration options are empty. This implies that the config options `classes_content` and `classes_content_header` will now affects the containers of `div.content` and `div.content-header` as previously did. Can you check it now?